### PR TITLE
Update ts_default_ods.xml

### PR DESCRIPTION
**Pull request by gho59**
Update with all Fields from table "x_timetracker_timesheet".
It would be nice to get the field "Title" instead of "timeaccount_id" , but I wasn't able to find out how.

### CODE REVIEW

File: tine20/Timetracker/Export/definitions/ts_default_ods.xml [hunk 1]
```diff
@@ -7,51 +7,144 @@
...
+            <identifier>Einsatzort</identifier>
+            <header>Einsatzort</header>
+            <type>string</type>
+            <width>2,5cm</width>
         </column>
-    </columns>
+  <column>
+    <identifier>invoice_id</identifier>
+	<header>invoice_id</header>
+    <Type>varchar(255)</Type>
```
Comment (msphn) [inline tine20/Timetracker/Export/definitions/ts_default_ods.xml]: Wrong indentation

Comment (msphn) [reply tine20/Timetracker/Export/definitions/ts_default_ods.xml]: also please write type in lower case

File: tine20/Timetracker/Export/definitions/ts_default_ods.xml [hunk 2]
```diff
@@ -7,51 +7,144 @@
...
+	<width>2,5cm</width>
+  </column>
+  <column>
+    <identifier>end_time</identifier>
+	<header>end_time</header>
+    <Type>time</Type>
+	<width>2,5cm</width>
```
Comment (msphn) [inline tine20/Timetracker/Export/definitions/ts_default_ods.xml]: didn't check it, but quite sure we wouldn't implement cm with a , as a decimal separator. were did you get this piece from? is it actually working?

Comment (gho59) [reply tine20/Timetracker/Export/definitions/ts_default_ods.xml]: It is working on 
Tine 2018.08.4 on Ubuntu 18.04 (repo: https://packages.tine20.org/ubuntu/ bionic)
the "type" is a copy'n paste from MySqlWorkbenchexport. excuse me I didn't take care of  casesensitives. Ijust tetsted it in my installation and it worked so far.

File: tine20/Timetracker/Export/definitions/ts_default_ods.xml [hunk 3]
```diff
@@ -7,51 +7,124 @@
...
-            <header>Billable</header>
-            <type>float</type>
-        </column>
-        <column>
-            <identifier>is_cleared_combined</identifier>
-            <header>Cleared</header>
-            <type>float</type>
-        </column>
-    </columns>
+   <columns>
```
Comment (msphn) [inline tine20/Timetracker/Export/definitions/ts_default_ods.xml]: indentation is still wrong

**Comment by msphn** [comment]
@gho59 I fixed the indendation now it's looking good.

@pschuele your turn!

File: tine20/Timetracker/Export/definitions/ts_default_ods.xml [hunk 4]
```diff
@@ -31,27 +26,105 @@
...
+            <header>deleted_time</header>
+            <type>datetime</type>
+        </column>
+        <column>
+            <identifier>seq</identifier>
+            <header>seq</header>
+            <type>int(11)</type>
+        </column>
+        <column>
+            <identifier>Einsatzort</identifier>
```
Comment (pschuele) [inline tine20/Timetracker/Export/definitions/ts_default_ods.xml]: seems to be a customfield and should be removed from the default export

File: tine20/Timetracker/Export/definitions/ts_default_ods.xml [hunk 5]
```diff
@@ -31,27 +26,105 @@
...
+            <header>invoice_id</header>
+            <type>varchar(255)</type>
+        </column>
+        <column>
+            <identifier>end_time</identifier>
+            <header>end_time</header>
+            <type>time</type>
         </column>
         <column>
-            <identifier>is_billable_combined</identifier>
```
Comment (pschuele) [inline tine20/Timetracker/Export/definitions/ts_default_ods.xml]: i would prefer that the Billable/Cleared properties are exported, too.

Comment (pschuele) [reply tine20/Timetracker/Export/definitions/ts_default_ods.xml]: still missing the combined fields.

**Comment by pschuele** [comment]
thanks for the PR! :)

**Pull request merged**